NGitLab/Models/CommitStatus.cs

```diff
@@ -1,9 +1,11 @@
-﻿using System.Text.Json.Serialization;
+﻿using System;
+using System.Text.Json.Serialization;
 
 namespace NGitLab.Models
 {
     public class CommitStatus
     {
+        [Obsolete("This is NOT the 'Project Id' (could be some kind of 'Commit Status Id')")]
```

Comment:
As it's not a ProjectId at all, could we:
- JsonIgnore ProjectId + obsolete
- Add the 'Id' property with corresponding JsonPropertyName + comment to specify that it's the Id of the commit status if needed